test/ui/confirm-test.js

```diff
@@ -17,128 +15,68 @@ for (let key in requirements) {
 // Hyper-V is not enabled on CI
 delete requirements.hyperv;
 
-const messages = {
-  detected: 'Using detected version',
-  older: 'Older than required!',
-  newer: 'Newer than required!'
-};
-
-describe('Confirm page', function confimPage() {
-  let confirmForm, detectionInfo, nextButton;
+describe('Confirmation page', function() {
+  let installsize, downloadSize, backButton, nextButton, cancelButton, browseButton, confirmNote, summaryHeading;
 
   beforeAll(function() {
-    browser.setLocation('confirm')
+    browser.setLocation('selection')
```

Comment:
If you want to run this test after selection has been done, you can just add it to https://github.com/redhat-developer-tooling/developer-platform-install/blob/master/protractor-conf.js#L30 after the selection test. It runs the files in sequence.

Reply:
yes will add the changes in the protractor-conf.js.